crates/rome_js_formatter/src/js/module/import_assertion_entry.rs

```diff
@@ -29,7 +30,7 @@ impl FormatNodeFields<JsImportAssertionEntry> for FormatNodeRule<JsImportAsserti
                 formatted_key,
                 colon_token.format(),
                 space_token(),
-                format_string_literal_token(value_token?, formatter),
+                FormatLiteralStringToken::from_string(&value_token?).format(formatter)?
```

Comment:
No need to call `format` here, `formatted` will do that for you
```suggestion
                FormatLiteralStringToken::from_string(&value_token?)
```